src/clandinin_lab_to_nwb/brezovec/brezovecbehaviorinterface.py

```diff
@@ -1,22 +1,245 @@
 """Primary class for converting experiment-specific behavior."""
+from pathlib import Path
 from pynwb.file import NWBFile
+from hdmf.backends.hdf5.h5_utils import H5DataIO
+from neuroconv.utils.json_schema import FolderPathType, FilePathType
+from neuroconv.tools.nwb_helpers import get_module
+from pynwb.behavior import SpatialSeries
+import polars as pl
+
 
 from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.utils import DeepDict
 
 class BrezovecBehaviorInterface(BaseDataInterface):
```

Comment:
Why not call this FictrackInterface?

Reply:
Yes, that makes sense. There is nothing specific here so far. I will do it.

Reply:
'FicTracInterface`, no K and capital T

from spelling on main package: https://github.com/rjdmoore/fictrac

Reply:
Done.